### PR TITLE
[12.0][MIG] report_csv

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration

@etobella

### CODE REVIEW

File: report_csv/report/report_csv.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 class ReportCSVAbstract(models.AbstractModel):
     _name = 'report.report_csv.abstract'
+    _description = 'report.report_csv.abstract'
```
Comment (etobella) [inline report_csv/report/report_csv.py]: Set a better name please

File: report_csv/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Enric Tobella <etobella@creublanca.es>
```
Comment (etobella) [inline report_csv/readme/CONTRIBUTORS.rst:1]: Add yourself

**Review by etobella — APPROVED** [review]
LGTM just some minor comments

**Comment by jarroyomorales** [comment]
@etobella Comments attended!

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
I just tested on runbot and locally and it is printing nothing for me.
Peek 2019-10-29 12-48

**Comment by jarroyomorales** [comment]
@lreficent Fixed! Sorry for the late reply, I didnt see the comment :sweat_smile:

File: report_csv/models/ir_report.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    @api.model
+    def _get_report_from_name(self, report_name):
+        res = super(ReportAction, self)._get_report_from_name(report_name)
+        if res:
+            return res
+        report_obj = self.env['ir.actions.report']
+        qwebtypes = ['csv']
+        import logging
+        logging.info(report_name)
```
Comment (LoisRForgeFlow) [inline report_csv/models/ir_report.py]: you probably added this to debug, should it be removed?

File: report_csv/models/ir_report.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        conditions = [('report_type', 'in', qwebtypes),
+                      ('report_name', '=', report_name)]
+        context = self.env['res.users'].context_get()
+        logging.info(report_obj.with_context(context).search(conditions, limit=1))
```
Comment (LoisRForgeFlow) [inline report_csv/models/ir_report.py]: same here

**Review by LoisRForgeFlow — APPROVED** [review]
Tested and works good now. A small comment inline.

Thanks! :+1:

File: report_csv/models/ir_report.py (hunk 3 above)
Comment (jarroyomorales) [reply report_csv/models/ir_report.py]: Yes sorry, just removed them

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**